scripts/hail_batch/hgdp1kg_tobwgs_pca/hgdp_1kg_tob_wgs_pca.py

```diff
@@ -25,6 +25,7 @@ def query(output):  # pylint: disable=too-many-locals
     hgdp_1kg = hl.read_matrix_table(GNOMAD_HGDP_1KG_MT)
     tob_wgs = hl.read_matrix_table(TOB_WGS).key_rows_by('locus', 'alleles')
     loadings = hl.read_table(GNOMAD_LIFTOVER_LOADINGS).key_by('locus', 'alleles')
+    mt_path = f'{output}/hgdp1kg_tobwgs_joined.mt'
```

Comment:
Is there a reason you pulled this so far up? Generally I'd recommend defining variables "as late as possible", this way it's easier to find everything relevant in close proximity of a particular context.

Reply:
I moved this down, just before checkpointing the merged matrix table